src/util/util.c

```diff
@@ -13,3 +14,42 @@ void _putchar(char character)
 {
     microkit_dbg_putc(character);
 }
+
+void print_bitarray(bitarray_t* bitarr) {
+#ifdef CONFIG_DEBUG_BUILD
```

Comment:
I'm confused why this is guarded.

Reply:
its so that these functions do nothing when not not compiling in debug build